flow/src/main/java/com/sequenceiq/flow/cleanup/FlowCleanupConfig.java

```diff
@@ -9,7 +9,14 @@ public class FlowCleanupConfig {
     @Value("${flowcleanup.intervalhours:24}")
     private int intervalInHours;
 
+    @Value("${flowcleanup.endtime.duration.hours:24}")
+    private int endTimeDurationInHours;
+
     public int getIntervalInHours() {
         return intervalInHours;
     }
+
+    public int getEndTimeDurationInHours() {
+        return endTimeDurationInHours;
+    }
 }
```

Comment:
Is there any reason for not using lombok to generate getters?

Reply:
I am not too sure myself. A core CB dev can probably answer this. cc @sodre90

Reply:
I think the biggest blocker will be setting up the IDE for Lombok to include the plugin and enable the annotation processors. Also with so many data models defined it may make it bulkier when you generate getters and setters for classes that dont require them, where the developer can selectively generate methods which are really needed.

Reply:
You only need to install plugin for lombok for setting up IDE. Also, it will not generate getters and setters for classes if you don't need them because you need to explicitly mention lombok annotations to generate getter,setter,toString etc.

Reply:
We try to avoid every byte code manipulation thing (unfortunately we use aspects in some cases...). IMHO it does not worth the technical dept it carries, if we want to move to another Java version we must wait for Lombok to support it. (Usually, it supports the new Java version in a month or something like this, so it is not a huge disatvantage)
But it is just a shortcut to create getters and setters, so writing them is not a big deal. Anyway, java 14 Records feature can be an alternative. We have an in-progress jira ticket to move to java 17 https://jira.cloudera.com/browse/CB-10310.

Reply:
Thanks @sodre90